awsapilib/controltower/controltower.py

```diff
@@ -45,6 +45,8 @@
 
 from opnieuw import retry
 from awsapilib.authentication import Authenticator, LoggerMixin
+from boto3_type_annotations.organizations import Client as organizationsClient
```

Comment:
Please use consistent class naming convention `OrganizationsClient` (full camel case for all class type annotations)